test/os_linux_generic.py

```diff
@@ -80,7 +80,7 @@ def test_get_mount_point_ext(self):
 
     def find_candidates_with_patch(self, mount_list, link_dict, listdir_dict):
         with patch('mbed_lstools.linux.MbedLsToolsLinuxGeneric._run_cli_process') as _cliproc,\
-             patch('mbed_lstools.linux.readlink') as _readlink,\
+             patch('mbed_lstools.linux.os.readlink') as _readlink,\
```

Comment:
Could this also be `os.readlink`? would that be more readable?

Reply:
Instead of `mbed_lstools.linux.os.readlink`

Reply:
Yeah sure, will do